captainhook_enabled.json

```diff
@@ -0,0 +1,62 @@
+{
+    "commit-msg": {
+        "enabled": true,
+        "actions": [
+            {
+                "action": "\\AppBundle\\GitHooks\\Action\\Regex",
+                "options": {
+                    "regex": "/^[A-Z]+-\\d+:?\\s/",
+                    "error": "Your commit message MUST start with a ticket ID"
+                }
+            }
+        ]
+    },
+    "pre-push": {
+        "enabled": false,
+        "actions": []
+    },
+    "pre-commit": {
+        "enabled": true,
+        "actions": [
+            {
+                "action": "\\AppBundle\\GitHooks\\Action\\CheckForBlacklistedWords",
+                "options": {
+                    "extensions": ["php", "twig"],
+                    "keyword_blacklist": ["var_dump", "dump"],
+                    "whitelisted_files": ["src/AppBundle/GitHooks/Action/CheckForBlacklistedWords.php"]
```

Comment:
This is excluded because of `$blacklistedKeywords = $action->getOptions()->get('keyword_blacklist', 'var_dump');` ?

Why do we need a default value in there?

Reply:
Done in 5e788b5e